socorro/external/postgresql/new_crash_source.py

```diff
@@ -0,0 +1,242 @@
+    def save_raw_crash_with_file_dumps(self, raw_crash, dumps, crash_id):
+        self._implementation.save_raw_crash_with_file_dumps(
+            raw_crash,
+            dumps,
+            crash_id
+        )
+
+    #--------------------------------------------------------------------------
+    def save_processed(self, processed_crash):
+        self._implementation.save_processed(processed_crash)
+
+    #--------------------------------------------------------------------------
+    def save_raw_and_processed(
+        self,
+        raw_crash,
+        dumps,
+        processed_crash,
+        crash_id
+    ):
+        self._implementation.save_raw_and_processed(
+            raw_crash,
+            dumps,
+            processed_crash,
+            crash_id
+        )
+
+    #--------------------------------------------------------------------------
+    def get_raw_crash(self, crash_id):
+        return self._implementation.get_raw_crash(crash_id)
+
+    #--------------------------------------------------------------------------
+    def get_raw_dump(self, crash_id, name=None):
+        return self._implementation.get_raw_dump(crash_id, name)
+
+    #--------------------------------------------------------------------------
+    def get_raw_dumps(self, crash_id):
+        return self._implementation.get_raw_dumps(crash_id)
+
+    #--------------------------------------------------------------------------
+    def get_raw_dumps_as_files(self, crash_id):
+        return self._implementation.get_raw_dumps_as_files(crash_id)
+
+    #--------------------------------------------------------------------------
+    def get_processed(self, crash_id):
+        return self._implementation.get_processed(crash_id)
+
+    #--------------------------------------------------------------------------
+    def get_unredacted_processed(self, crash_id):
+        return self._implementation.get_unredacted_processed(crash_id)
+
+    #--------------------------------------------------------------------------
+    def remove(self, crash_id):
+        self._implementation.remove(crash_id)
+
+
+#==============================================================================
+class PGPVNewCrashSource(PGQueryNewCrashSource):
+    required_config = Namespace()
+    required_config.crash_id_query = change_default(
+        PGQueryNewCrashSource,
+        'crash_id_query',
+        "select uuid from reports "
+        "where ( "
+        "date_processed >= %s and date_processed < %s "
+        "and ({pv_where_clause}))"
+    )
+    required_config.add_option(
+        "product_version_list",
+        doc='list of the form "p1, v1a, v1b; p2, v2a, v2b; ..."',
+        default='Firefox',
+        from_string_converter=two_level_nested_list_converter(),
+        likely_to_be_changed=True
+    )
+    required_config.add_option(
+        'date',
+        doc="a date in the form YYYY-MM-DD",
+        default=(utc_now() - timedelta(1)).date(),
```

Comment:
Again, why is this configurable? It should just be a passing parameter. 
Having a default that freezes when you import the code just feels wrong. 
I've seen this in other places too and it makes it cringe ever time. 

If it really has to be a config value, can the default at least be `default=lambda: =(utc_now() - timedelta(1)).date()`